ui/helpers/utils/token-util.js

```diff
@@ -80,23 +78,12 @@ async function getDecimals(tokenAddress) {
   return decimals;
 }
 
-export async function getSymbolAndDecimals(tokenAddress, existingTokens = []) {
```

Comment:
Was `existingTokens` not used anywhere?

Reply:
`getSymbolAndDecimals` is used only in two locations in `tokenInfoGetter` in `tokens-util` and `getTokenParams` in `actions` and both of them seems to be returning the token info if it is an existingToken.